test/powershell/Modules/Microsoft.PowerShell.Core/TestGetCommand.Tests.ps1

```diff
@@ -211,7 +211,7 @@
         $paramName = "Encoding"
         $results = get-command -verb get -noun content -Encoding Unicode
```

Comment:
Now that you have touched the file. Can you fix the casing for the Get-Command cmdlets. 
get-command -> Get-Command 
-verb -> -Verb
-noun -> -Noun

Reply:
done